### PR TITLE
Calorie progress bar initial commit

### PR DESCRIPTION
**Pull request by mmoren01**
Adding the calorie progress bar as is for demo purposes. To Do's: Hook it up to live data

### CODE REVIEW

File: components/CalorieTracker.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, {useState} from 'react';
+import Apple from '../public/appleStencil.svg';
+
+import ProgressBar from 'react-bootstrap/ProgressBar';
+
+const CalorieTracker = () => {
+  const [progress, setProgress] = useState(60);
```
Comment (emmaknor) [inline components/CalorieTracker.jsx]: why is this initial state set to 60?

File: components/dailyTracker.jsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import styles from '../styles/Home.module.css';
+import CalorieTracker from './CalorieTracker.jsx';
 
 const DailyTracker = () => {
   return (
     <>
       <div className={`${styles.card} daily-tracker`}>
         <p className={`${styles.description} subtitle`}>{`Daily Tracker`}</p>
+        <CalorieTracker />
```
Comment (emmaknor) [inline components/dailyTracker.jsx:18]: we may have some merge conflicts here, but we can sort it out!

**Review by benkennedy98 — COMMENTED** [review]
Looks good!

File: pages/_app.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 /* eslint-disable react/prop-types */
 /* eslint-disable require-jsdoc */
 import '../styles/globals.css';
+import '../styles/calorie-progress-bar.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (emmaknor) [inline pages/_app.js:5]: another potential merge conflict

File: public/appleStencil.svg [hunk 4]
```diff
@@ -0,0 +1 @@
... 164.400 66.948,161.892 69.940,155.124 72.851 C 146.851 76.410,138.090 77.311,117.570 76.710 C 95.179 76.054,88.028 74.872,60.783 67.326 C 38.174 61.064,20.289 57.030,15.000 56.999 C 13.075 56.987,9.836 56.703,7.802 56.367 L 4.104 55.755 8.975 48.537 C 29.370 18.314,81.006 -3.727,114.303 3.579 M254.744 58.414 C 267.280 61.765,273.982 67.152,283.603 81.612 C 294.430 97.883,298.449 111.838,298.404 133.000 C 298.360 154.087,293.979 168.760,281.548 189.459 C 273.838 202.296,253.872 228.935,244.500 238.889 C 235.731 248.202,224.878 255.468,217.426 257.015 C 210.987 258.352,207.551 257.918,192.000 253.801 C 179.318 250.445,178.443 250.480,162.000 255.001 C 149.832 258.348,149.692 258.290,126.840 240.410 C 103.786 222.372,89.450 202.569,72.249 165.000 C 64.208 147.438,62.666 141.350,62.751 127.500 C 62.862 109.611,68.337 93.454,78.632 80.637 L 83.341 74.775 87.920 75.431 C 101.635 77.397,110.064 77.986,124.590 77.992 C 143.258 78.001,153.618 76.002,160.746 71.017 C 164.291 68.538,165.417 68.208,167.561 69.023 C 173.825 71.405,179.316 70.264,192.364 63.870 L 204.874 57.741 217.687 56.958 C 233.452 55.994,247.801 56.558,254.744 58.414 " stroke="none" fill="white" fill-rule="evenodd"/></svg>
```
Comment (emmaknor) [inline public/appleStencil.svg:1]: is this all one line? looks like some lint to me

**Review by emmaknor — APPROVED** [review]

File: components/CalorieTracker.jsx (hunk 1 above)
Comment (mmoren01) [reply components/CalorieTracker.jsx]: For testing. Will be set to zero once we start pulling data.

File: components/dailyTracker.jsx (hunk 2 above)
Comment (mmoren01) [reply components/dailyTracker.jsx:18]: Will address this once the previous PRs are approved

File: pages/_app.js (hunk 3 above)
Comment (mmoren01) [reply pages/_app.js:5]: Same as last comment. Will address when previous PR's are merged

File: public/appleStencil.svg (hunk 4 above)
Comment (mmoren01) [reply public/appleStencil.svg:1]: It's an image file from a compiler. Should be okay to disregard. Please let me know if this needs to be addressed

**Review by nohren — COMMENTED** [review]
Nice work!

**Review by nohren — APPROVED** [review]

**Pull request merged**